### PR TITLE
201810 curator pages

### PR DESCRIPTION
**Pull request by madoleary**
* [x] Are you working on the latest release?
* [x] Have you tested it locally?
* [->] Please explain your change

This separates out the home page into distinct pages, as I found the current layout, particularly for curators, super confusing. There's now separate pages for changes requested, points pending review, and drafts. The tab functionality is still rudimentary but can be improved upon in later iterations.

I would like to clarify a few things as part of this pull request:
- Users who are curators can view changes requested, points pending review and drafts?
- Users who are not curators can only view drafts and changes requested?

Thanks !

### CODE REVIEW

**Comment by Vinnl** [comment]
Separating them out into different pages sounds fine to me, but I think it would be good if the homepage did show the thing that is more relevant to the current user, rather than first having to click through to a different tab.

In other words:

- If some of your points have changes-requested, show those - those are usually most pressing. Note that for most users, there are only a few points in this state, if any; there's only a lot of them for you since you adopted a lot from the import script :) Usually, you create a number of points, and maybe one or two need another look.
- Otherwise, if you're a curator, and there are still points that need reviewing (which should eventually hopefully be none, or just a few that have just been created), show those, as I think that is the most important job curators can do.
- Otherwise, if you still have points in draft, you'll probably want to finish those, so show those.

(There's something to be said for swapping the last two around as well.)

Ideally, the tabs would only show if the number of items in them is more than zero.

Basically, my current workflow is: visit the homepage of edit.tosdr.org, and immediately see what work there is for me to do. With the change in its current state, that changes to:

- go to the homepage
- click on Change Requests
- if none are listed there, click on Pending points
- If none are listed there, click on Drafts

In other words, that's four times waiting for a page to load and having to scan that page, rather than one :)

**Comment by madoleary** [comment]
Ah, ok! Thanks for the feedback. I'll work from it.

**Comment by JesseWeinstein** [comment]
Is this PR so stale that it should be closed?

**Comment by michielbdejong** [comment]
@madoleary can you check?
